### PR TITLE
Null-checks added.

### PR DESCRIPTION
**Pull request by Azoraqua**
+ To make sure no NullPointerException will ever happen.

### CODE REVIEW

File: src/com/mrcrayfish/modelcreator/Settings.java [hunk 1]
```diff
@@ -12,53 +12,71 @@
...
+			prefs.put(JSON_DIR, dir);
+		}
 	}		
 	
 	private static Preferences getPreferences()
 	{
-		return Preferences.userNodeForPackage(Settings.class);
+		Preferences prefs = Preferences.userNodeForPackage(Settings.class);
+		
+		return (prefs != null ? prefs : null);
```
Comment (KnightMiner) [inline src/com/mrcrayfish/modelcreator/Settings.java]: This is a useless check, if `prefs` is null it is already returning null.

**Comment by Azoraqua** [comment]
If 'prefs' is null, a NullPointerException might occur when trying to get data from the 'prefs' reference. 

Correct me if I am wrong, still learning haha :)

**Comment by MrCrayfish** [comment]
Think about it, `prefs` could either be an instance or null. Why do we need a second check when we already know and can simply return `prefs`. I'm struggling to make this simpler than @KnightMiner did already.

**Comment by Azoraqua** [comment]
@MrCrayFish, Oh okay. We have different idea's of goals I see. But that's always nice to see.  Simplified and useful, great :)

**Pull request closed without merging**